js/modules/privacy.js

```diff
@@ -1,34 +1,63 @@
 /* eslint-env node */
 
-const Path = require('path');
+const is = require('@sindresorhus/is');
+const path = require('path');
 
-const {
-  escapeRegExp,
-  isRegExp,
-  isString,
-} = require('lodash');
 const { compose } = require('lodash/fp');
+const { escapeRegExp } = require('lodash');
 
 
+const APP_ROOT_PATH = path.join(__dirname, '..', '..', '..');
 const PHONE_NUMBER_PATTERN = /\+\d{7,12}(\d{3})/g;
 const GROUP_ID_PATTERN = /(group\()([^)]+)(\))/g;
+const REDACTION_PLACEHOLDER = '[REDACTED]';
+
+
+//      _redactPath :: Path -> String -> String
+exports._redactPath = (filePath) => {
+  if (!is.string(filePath)) {
+    throw new TypeError('"filePath" must be a string');
```

Comment:
It'll be good to make that eslint change so we can move to `"'filePath' must be a string"`

Reply:
Coming up in another PR.